assets/tests/components/detours/diversionPage.test.tsx

```diff
@@ -33,14 +33,15 @@ import { ok, loading } from "../../../src/util/fetchResult"
 const DiversionPage = (
   props: Omit<
     Partial<ComponentProps<typeof DiversionPageDefault>>,
-    "originalRoute"
+    "originalRoute" | "showConfirmCloseModal"
   > & {
     originalRoute?: Partial<
       ComponentProps<typeof DiversionPageDefault>["originalRoute"]
     >
+    showConfirmCloseModal?: boolean
```

Comment:
question(non-blocking): is this needed? it seems like the `Partial` on ln35 would already do this. The override on ln38 is needed because `Partial` doesn't imply `DeepPartial`, but it seems like that wouldn't matter for this type?

Reply:
Ah I went back and forth a bunch getting this type to work, but you're right that it can be simplified. See 0c4e8865.